akka-docs/src/main/paradox/scala/cluster-team.md

```diff
@@ -1,15 +1,109 @@
+When @ref:[subscribing to cluster events](cluster-usage.md#cluster-subscriber) the `UnreachableMember` and
+`ReachableMember` events are for observations within the own team. The same team as where the
+subscription was registered.
+
+For cross team unreachability notifications you can subscribe to `UnreachableTeam` and `ReachbleTeam`
+events.
+
+## Cluster Singleton
+
+The [Cluster Singleton](cluster-singleton.md) is a singleton per team. If you start the 
+`ClusterSingletonManager` on all nodes and you have defined 3 different teams there will be
+3 active singleton instances in the cluster, one in each team. This is taken care of automatically, 
+but is important to be aware of.
+
+The reason why the singleton is per team and not global is that there is not enough consistency
+in the membership management when using one leader per team to select a single global singleton.
+
+If you need a global singleton you have to pick one team to host that singleton and only start the
+`ClusterSingletonManager` on nodes of that team. If the team is unavailable from another team the
+singleton is unavailable, which is a reasonable trade-off when selecting a design for consistency
+over availability.
+
+The `ClusterSingletonProxy` is by default routing messages to the singleton in the own team, but
+it can be started with a `team` parameter in the `ClusterSingletonProxySettings` to define that 
+it should route messages to a singleton located in another team. That is useful for example when
+having a global singleton in one team and accessing it from other teams.
+
+## Cluster Sharding
+
+The coordinator in [Cluster Sharding](cluster-sharding.md) is a Cluster Singleton and therefore,
+as explained above, Cluster Sharding is also per team. Each team will have its own coordinator
+and regions, isolated from other teams. If you start an entity type with the same name on all 
+nodes and you have defined 3 different teams and then send messages to the same entity id to
+sharding regions in all teams you will end up with 3 active entity instances for that entity id,
+one in each team. 
+
+Especially when used together with Akka Persistence that is based on the single-writer principle
+it is important to avoid running the same entity at multiple locations at the same time with a
+shared data store. Lightbend's Akka Team is working on solution that will support multiple active
+entities that will work nicely together with Cluster Sharding across multiple data centers.
+
+If you need global entities you have to pick one team to host that entity type and only start
+`ClusterSharding` on nodes of that team. If the team is unavailable from another team the
+entities are unavailable, which is a reasonable trade-off when selecting design for consistency
+over availability.
+
+The Cluster Sharding proxy is by default routing messages to the shard regions in the own team, but
+it can be started with a `team` parameter to define that it should route messages to a singleton
```

Comment:
Shouldn't that read "should route message to a *shard region* located in another team"?